system-parachains/encointer/src/lib.rs

```diff
@@ -77,7 +77,7 @@ pub use parachains_common::{
 };
 use polkadot_runtime_common::{BlockHashCount, SlowAdjustingFeeUpdate};
 use sp_api::impl_runtime_apis;
-use sp_core::{crypto::KeyTypeId, ConstU32, OpaqueMetadata};
+use sp_core::{crypto::KeyTypeId, ConstU32, ConstU64, OpaqueMetadata};
```

Comment:
do we need `ConstU64`?

Reply:
It was another error I got, but unrelated to the FungibleAdapter thing, I can remove it

Reply:
yes, please, also CI complains about that: https://github.com/bkontur/runtimes/actions/runs/7905400214/job/21584484750?pr=1